src/modules/emote_menu/twitch/EmoteMenu.jsx

```diff
@@ -100,7 +100,7 @@ function unloadButton(container, nativeEmotePickerButton) {
     nativeEmotePickerButton = document.querySelector(EMOTE_PICKER_BUTTON_SELECTOR);
   }
   if (nativeEmotePickerButton != null) {
-    nativeEmotePickerButton.classList.remove(styles.hideEmoteMenuButton);
+    nativeEmotePickerButton.parentElement.classList.remove(styles.hideEmoteMenuButtonContainer);
```

Comment:
we should make some sort of assertion that the parentElement has specific properties to ensure we don't accidentally hide a container with multiple other things.

Reply:
The css queries for the button, so hopefully this won't happen.

Reply:
Maybe it would be wiser to apply the style to the chat input button container?

Reply:
i dont know if there's an easy way to target that container. we could target the text area by class name, and then this emote button by target name.